pkg/reconciler/trigger/resources/subscription.go

```diff
@@ -67,6 +67,7 @@ func NewSubscription(t *eventingv1alpha1.Trigger, brokerTrigger, brokerIngress *
 					},
 				},
 			},
+			Delivery: b.Spec.Delivery,
```

Comment:
since you only need the delivery here, just curious why not just pass in that part of the spec.

Reply:
What do you mean? As part of the trigger spec instead of the broker spec?

Reply:
as part of the function signature